lib/jsonld-signatures.js

```diff
@@ -57,7 +57,7 @@ api.suites = require('./suites').suites;
 api.purposes = require('./purposes').purposes;
 
 // expose LDKeyPair classes
-Object.assign(api, require('./LDKeyPair'));
+// Object.assign(api, require('./LDKeyPair'));
```

Comment:
We need to expose the keypair stuff to avoid doing another major release.

```suggestion
  Object.assign(api, require('crypto-ld'));
```